tests/storage/business/test_arealink_manager.py

```diff
@@ -122,205 +122,210 @@ def test_area_crud(empty_study: FileStudy, matrix_service: SimpleMatrixService):
     )
     assert len(empty_study.config.areas.keys()) == 0
 
-    area_manager.create_area(study, AreaCreationDTO(name="test", type=AreaType.AREA))
-    assert len(empty_study.config.areas.keys()) == 1
-    assert json.loads((empty_study.config.study_path / "patch.json").read_text())["areas"]["test"]["country"] is None
-
-    area_manager.update_area_ui(study, "test", UpdateAreaUi(x=100, y=200, color_rgb=(255, 0, 100)))
-    assert empty_study.tree.get(["input", "areas", "test", "ui", "ui"]) == {
-        "x": 100,
-        "y": 200,
-        "color_r": 255,
-        "color_g": 0,
-        "color_b": 100,
-        "layers": 0,
-    }
-
-    area_manager.create_area(study, AreaCreationDTO(name="test2", type=AreaType.AREA))
-
-    link_manager.create_link(
-        study,
-        LinkDTO(
-            area1="test",
-            area2="test2",
-        ),
-    )
-    assert empty_study.config.areas["test"].links.get("test2") is not None
+    with patch("antarest.study.business.utils.get_current_user", return_value=DEFAULT_ADMIN_USER):
```

Comment:
Note that if we create a context manager to set the current user (in task service), we will be more difficulte to maintain